HistContext/HistContext.gpr.py

```diff
@@ -0,0 +1,18 @@
+register(GRAMPLET,
+         id="HistContext", 
+         name=_("Historical Context"),
+         description = _("Displaying a historical context"),
+         status = BETA,
+         version = '0.2.10',
+         fname="HistContext.py",
+         height = 20,
+         detached_width = 510,
+         detached_height = 480,
+         expand = True,
+         gramplet = "HistContext",
+         gramplet_title=_("Historical Context"),
+         gramps_target_version="5.2",
+         help_url="https://github.com/kajmikkelsen/TimeLineGramplet",
```

Comment:
change this to point to the new help page on the Gramps wiki for this addon say

help_url="https://gramps-project.org/wiki/index.php/Addon:HistoricalContext",